xarray/tests/test_plot.py

```diff
@@ -88,6 +88,13 @@ def easy_array(shape, start=0, stop=1):
     return a.reshape(shape)
 
 
+def get_colorbar_label(colorbar):
+    if colorbar.orientation == "vertical":
+        return colorbar.ax.get_ylabel()
+    else:
+        return colorbar.ax.get_xlabel()
```

Comment:
it might make sense to push this upstream as `colorbar.get_label()`

Reply:
Agreed!